tests/scripts/all.sh

```diff
@@ -3505,6 +3505,11 @@ component_build_armcc () {
     # ARM Compiler 6 - Target ARMv8.2-A - AArch64
     armc6_build_test "-O1 --target=aarch64-arm-none-eabi -march=armv8.2-a+crypto"
 }
+support_build_armcc () {
+    arm5_cc="$ARMC5_BIN_DIR/armcc"
+    arm6_cc="$ARMC6_BIN_DIR/armclang"
```

Comment:
These are already defined as `$ARM5_CC` and `$ARM6_CC`.

Reply:
Sorry for misleading. I guess you mean `$ARMC5_CC` andf `$ARMC6_CC`.
Yes there are, but at the time the `support_xxx` functions are excuted, `ARMC5_CC` and `ARMC6_CC` is still unbound.